py/picca/delta_extraction/utils.py

```diff
@@ -82,8 +82,10 @@
     "LY10": 919.3514,
 }
 
-ACCEPTED_BLINDING_STRATEGIES = ["none", "minimal", "corr_yshift"]
-
+ACCEPTED_BLINDING_STRATEGIES = [
+    "none", "minimal", "desi_m2", "desi_y1", "desi_y3"]
```

Comment:
When/where is the "minimal" strategy used? Should we remove it?

Reply:
I guess we can remove it, given that we also removed the corr_yshift and that we are not allowing the user to choose